### DISCUSSION TITLE
X1 OMNI - Trying to restart from scratch from your fork

### DISCUSSION THREAD

**Discussion by WhynotThomas** [MVladislav/bumper#55] [Q&A]
Hello, 

Great thx for your work here !

i heard that your work solved some X1 issues, im trying to start new project on Docker with your branch, 
no idea about how do i have to start? 

do i need to pull the Martin branch than use the configs files in from your git or do you have any pull folded that i have to use? 

Do you have any tutorial to use your work? 

Like, Pull and build link, certs to uses and config files?

i'm on Raspberry PI 4B 8gb Ram, with Router (DNS fully handled) but .. i'm lost again.

ty :)

**Comment by MVladislav** [comment]
Hi,

Just to clarify, are you looking to only start the project locally without developing on it? Please let me know if I understand correctly.

To get the project up and running, the README contains some helpful information. I've updated it today. In general, you can start the project by running the following commands:

```sh
git clone https://github.com/MVladislav/bumper && cd bumper
echo "BUMPER_ANNOUNCE_IP=$(hostname -I | awk '{print $1}')" > .env
bash ./scripts/create_cert.sh
docker-compose up -d
```

- The certificate script creates a self-signed certificate that you can use.
- The server is accessible only by its hostname, not by IP. Therefore, you'll need to map the IP to the defined hostname.
